src/components/accountWizard/accountWizardStepResults.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Spinner } from 'patternfly-react';
+import { connect } from '../../redux';
+import apiTypes from '../../constants/apiConstants';
+
+const AccountWizardStepResults = ({ accountName, edit, error, errorMessage, fulfilled, pending }) => (
+  <React.Fragment>
+    {error && (
+      <div className="wizard-pf-complete blank-slate-pf">
+        <div className="wizard-pf-success-icon">
+          <span className="pficon pficon-error-circle-o" />
```

Comment:
Could use the pf-react `Icon` component here. It would add the aria-hidden tag.

Reply:
Can do!